pages/linux/rusnapshot.md

```diff
@@ -0,0 +1,20 @@
+# rusnapshot
+
+> BTRFS snapshotting utility made in rust.
+> For more information: <https://github.com/Edu4rdSHL/rusnapshot>.
+
+- Create a snapshot using a config file:
+
+`sudo rusnapshot -c {{/path/to/config.toml}} --cr`
```

Comment:
Is there a long variant of `-c`, `-l`, or `-k`?

Reply:
Yes, for -l is --list, for -k is --keep and for -c is --config.

Reply:
Nice! then @danyeet, let's replace the short options with the long options.  In case you don't know, we prefer the long options to the short ones because since they're normally English words, they're essentially self-explanatory, thus providing documentation in addition to that in the example description.  Makes sense?
 ```suggestion
`sudo rusnapshot --config {{/path/to/config.toml}} --cr`
```

Reply:
Yeah, fully agree, just to be honest I had to double check if there were long options in the first place :D.